pkgs/development/python-modules/astropy/default.nix

```diff
@@ -32,14 +33,18 @@ buildPythonPackage {
     hash = "sha256-9q4noHf46oSQPvp2x5C5hWFzQaAISw0hw5H3o/MyrCM=";
   };
 
-  SETUPTOOLS_SCM_PRETEND_VERSION = version;
```

Comment:
Is it OK to remove this?

Reply:
Thank you for checking with me. Yes, it should be. I verified that the wheel that's created has the version number as part of the `distinfo` directory name, and from what I have seen I think `setuptools-scm` does not need the pretend version when we fetch the source from PyPI. I think it's because that archive has a `PKG_INFO` file which it can use.

Reply:
That's good to know!
Thanks.